packages/libretro/libretro-database/package.mk

```diff
@@ -19,7 +19,7 @@
 ################################################################################
 
 PKG_NAME="libretro-database"
-PKG_VERSION="e23ab29"
+PKG_VERSION="5809b27"
```

Comment:
A few database updates...

``` suggestion
PKG_VERSION="2a8e537"
```

Reply:
I did run the update script a week ago, thanks for the heads up